### ISSUE TITLE
ON HOLD: [Med] Simplify DOM and CSS III

### ISSUE THREAD

**Issue by Larisa-Staroverova** [SAP/spartacus#9483]
Depends on #9176, #9177 

`configurator-issues-notification`,
`configure-cart-entry`,
`configure-product` and a part in `cart-item` where the inline configuration for cart item is added.

Pay attention to the point collected on the [Clean up DOM](https://wiki.wdf.sap.corp/wiki/display/CPQ/Clean+up+DOM) page
